common/styles/scss/_navbar.scss

```diff
@@ -163,29 +165,42 @@ body > .container {
     color: #fff;
 }
 
-
-// Horizontal navbar
-@media (min-width: 768px) {
-    .child-opened {
-        background-color: $table-header-background-color;
-    }
+.dropdown-menu > li >a {
```

Comment:
Styling changes to center the arrow, allow text wrap and add a cursor pointer to make it appear clickable.

Before 

![image](https://user-images.githubusercontent.com/78573171/120389345-28a0f780-c2e1-11eb-829e-7a28be264495.png)


After

![image](https://user-images.githubusercontent.com/78573171/120389509-5d14b380-c2e1-11eb-8488-664a60581647.png)